docs/source/faq.rst

```diff
@@ -0,0 +1,21 @@
+Frequently Asked Questions
+==========================
+
+.. contents:: The following is a list of common questions asked about pyOpenMS.
+
+How can I wrap a new method with pyOpenMS?
+------------------------------------------
+
+Add an entry to ``src/pyOpenMS/pxds/CLASS_NAME.pxd`` with the signature of your new method(s).
+
+
+How can I wrap a new class with pyOpenMS?
+-----------------------------------------
+
+Create a new file ``src/pyOpenMS/pxds/CLASS_NAME.pxd`` and use the `procedure outlined <https://pyopenms.readthedocs.io/en/latest/wrap_classes.html?#how-pyopenms-wraps-python-classes>`_. 
```

Comment:
i was not clear: let's link it to: wrap_classes.rst#how-to-wrap-new-classes

Reply:
done